app/assets/stylesheets/application.css.scss

```diff
@@ -80,6 +80,32 @@ input {
  margin-left: -17px;
 }
 
+.clearfix {
+  overflow: auto;
+}
+
+#results {
+  padding: 10px;
```

Comment:
Properties should be ordered background, border, border-color, border-radius, padding